dev/archery/archery/integration/datagen.py

```diff
@@ -1075,12 +1095,27 @@ def _temp_path():
         generate_primitive_case([], name='primitive_no_batches'),
         generate_primitive_case([17, 20], name='primitive'),
         generate_primitive_case([0, 0, 0], name='primitive_zerolength'),
-        generate_decimal_case(),
+
+        generate_null_case([10, 0])
+        .skip_category('JS')
```

Comment:
We should make Jiras for these skips. Also it appears that Java is not implemented (the test fails) so we should skip Java too.

Reply:
```suggestion
        .skip_category('Java') # TODO(ARROW-7899)
        .skip_category('JS') # TODO(ARROW-7900)
```

Reply:
ARROW-7899 is fixed now.